### ISSUE TITLE
Inviting users to a room causes LastPass to ask if I want to save the password

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-android#2686]
There's no password involved here. There's probably a flag or something that can be added to the invite field to avoid LastPass (and other password managers) from helping.

**Comment by bmarty** [comment]
@turt2live I cannot reproduce the issue. I've installed LastPass and ... Due you still get it on Riot v0.8.20?

**Comment by turt2live** [comment]
Yes. It looks like it is trying to pick accounts that I've previously saved when logging in.

Steps might look like:
1. Install LastPass and set it up
2. Log in to Riot and save the credentials in LastPass when prompted
3. Pick a room and tap the header
4. Tap the + icon to start inviting someone
5. Put your cursor in the field to receive the LastPass prompt

**Comment by bmarty** [comment]
I've done this recipe but wasn't prompted to save credentials at step 2. Will try again later
